security/docs/src/main/doc/IntegratingSecurity.adoc

```diff
@@ -1,5 +1,5 @@
 = Integrating {branding} Security Application
-include::../includes/config.adoc[]
+include:: ../../classes/config.adoc[]
```

Comment:
http://asciidoctor.org/docs/user-manual/#include-files-relative-to-a-common-source-directory

Did you try that option?